{
  "records": {
    "2018-08-26T00:00:00": [
      {
        "count": 1,
        "date": "2018-08-26T00:01:41.635245",
        "exc": "",
        "levelname": "INFO",
        "lineno": 164,
        "msg": "Automated data push.",
        "pathname": "/home/pi/orangeandblueleague/filefairy/plugin/git/git.py"
      },
      {
        "count": 1,
        "date": "2018-08-26T15:20:21.761289",
        "exc": "",
        "levelname": "INFO",
        "lineno": 204,
        "msg": "Tracker locked.",
        "pathname": "/home/pi/orangeandblueleague/filefairy/plugin/exports/exports.py"
      },
      {
        "count": 1,
        "date": "2018-08-26T16:11:09.570849",
        "exc": "",
        "levelname": "INFO",
        "lineno": 96,
        "msg": "Upload started.",
        "pathname": "/home/pi/orangeandblueleague/filefairy/plugin/leaguefile/leaguefile.py"
      },
      {
        "count": 1,
        "date": "2018-08-26T17:28:30.639187",
        "exc": "",
        "levelname": "INFO",
        "lineno": 294,
        "msg": "File is up.",
        "pathname": "/home/pi/orangeandblueleague/filefairy/plugin/leaguefile/leaguefile.py"
      },
      {
        "count": 1,
        "date": "2018-08-26T17:28:34.187454",
        "exc": "",
        "levelname": "INFO",
        "lineno": 288,
        "msg": "Download started.",
        "pathname": "/home/pi/orangeandblueleague/filefairy/plugin/leaguefile/leaguefile.py"
      },
      {
        "count": 1,
        "date": "2018-08-26T17:35:53.454712",
        "exc": "",
        "levelname": "INFO",
        "lineno": 270,
        "msg": "Download finished.",
        "pathname": "/home/pi/orangeandblueleague/filefairy/plugin/leaguefile/leaguefile.py"
      },
      {
        "count": 1,
        "date": "2018-08-26T17:35:57.584051",
        "exc": "",
        "levelname": "INFO",
        "lineno": 60,
        "msg": "News updated.",
        "pathname": "/home/pi/orangeandblueleague/filefairy/plugin/recap/recap.py"
      }
    ],
    "2018-08-27T00:00:00": [
      {
        "count": 1,
        "date": "2018-08-27T00:00:23.338357",
        "exc": "",
        "levelname": "INFO",
        "lineno": 164,
        "msg": "Automated data push.",
        "pathname": "/home/pi/orangeandblueleague/filefairy/plugin/git/git.py"
      },
      {
        "count": 2,
        "date": "2018-08-27T14:47:35.379920",
        "exc": "",
        "levelname": "INFO",
        "lineno": 247,
        "msg": "Reloaded exports.",
        "pathname": "/home/pi/orangeandblueleague/filefairy/core/fairylab/fairylab.py"
      },
      {
        "count": 1,
        "date": "2018-08-27T14:47:16.626446",
        "exc": "",
        "levelname": "INFO",
        "lineno": 155,
        "msg": "Fetched latest changes.",
        "pathname": "/home/pi/orangeandblueleague/filefairy/plugin/git/git.py"
      },
      {
        "count": 1,
        "date": "2018-08-27T14:48:17.579733",
        "exc": "",
        "levelname": "INFO",
        "lineno": 247,
        "msg": "Reloaded subprocess_.",
        "pathname": "/home/pi/orangeandblueleague/filefairy/core/fairylab/fairylab.py"
      }
    ],
    "2018-08-28T00:00:00": [
      {
        "count": 1,
        "date": "2018-08-28T00:01:07.584841",
        "exc": "",
        "levelname": "INFO",
        "lineno": 164,
        "msg": "Automated data push.",
        "pathname": "/home/pi/orangeandblueleague/filefairy/plugin/git/git.py"
      },
      {
        "count": 1,
        "date": "2018-08-28T15:46:07.771830",
        "exc": "",
        "levelname": "INFO",
        "lineno": 204,
        "msg": "Tracker locked.",
        "pathname": "/home/pi/orangeandblueleague/filefairy/plugin/exports/exports.py"
      },
      {
        "count": 1,
        "date": "2018-08-28T19:58:26.427434",
        "exc": "",
        "levelname": "INFO",
        "lineno": 96,
        "msg": "Upload started.",
        "pathname": "/home/pi/orangeandblueleague/filefairy/plugin/leaguefile/leaguefile.py"
      },
      {
        "count": 1,
        "date": "2018-08-28T21:04:37.967926",
        "exc": "",
        "levelname": "INFO",
        "lineno": 294,
        "msg": "File is up.",
        "pathname": "/home/pi/orangeandblueleague/filefairy/plugin/leaguefile/leaguefile.py"
      },
      {
        "count": 1,
        "date": "2018-08-28T21:04:41.758400",
        "exc": "",
        "levelname": "INFO",
        "lineno": 288,
        "msg": "Download started.",
        "pathname": "/home/pi/orangeandblueleague/filefairy/plugin/leaguefile/leaguefile.py"
      },
      {
        "count": 1,
        "date": "2018-08-28T21:16:34.358749",
        "exc": "",
        "levelname": "INFO",
        "lineno": 270,
        "msg": "Download finished.",
        "pathname": "/home/pi/orangeandblueleague/filefairy/plugin/leaguefile/leaguefile.py"
      },
      {
        "count": 1,
        "date": "2018-08-28T21:16:38.805719",
        "exc": "",
        "levelname": "INFO",
        "lineno": 60,
        "msg": "News updated.",
        "pathname": "/home/pi/orangeandblueleague/filefairy/plugin/recap/recap.py"
      }
    ],
    "2018-08-29T00:00:00": [
      {
        "count": 1,
        "date": "2018-08-29T00:01:02.662154",
        "exc": "",
        "levelname": "INFO",
        "lineno": 164,
        "msg": "Automated data push.",
        "pathname": "/home/pi/orangeandblueleague/filefairy/plugin/git/git.py"
      },
      {
        "count": 2,
        "date": "2018-08-29T11:57:17.033642",
        "exc": "",
        "levelname": "INFO",
        "lineno": 155,
        "msg": "Fetched latest changes.",
        "pathname": "/home/pi/orangeandblueleague/filefairy/plugin/git/git.py"
      },
      {
        "count": 1,
        "date": "2018-08-29T11:56:28.217176",
        "exc": "",
        "levelname": "INFO",
        "lineno": 164,
        "msg": "Manual data push.",
        "pathname": "/home/pi/orangeandblueleague/filefairy/plugin/git/git.py"
      },
      {
        "count": 1,
        "date": "2018-08-29T11:57:27.157377",
        "exc": "",
        "levelname": "INFO",
        "lineno": 247,
        "msg": "Reloaded exports.",
        "pathname": "/home/pi/orangeandblueleague/filefairy/core/fairylab/fairylab.py"
      }
    ],
    "2018-08-30T00:00:00": [
      {
        "count": 1,
        "date": "2018-08-30T00:00:45.841964",
        "exc": "",
        "levelname": "INFO",
        "lineno": 164,
        "msg": "Automated data push.",
        "pathname": "/home/pi/orangeandblueleague/filefairy/plugin/git/git.py"
      },
      {
        "count": 1,
        "date": "2018-08-30T16:56:48.080841",
        "exc": "",
        "levelname": "INFO",
        "lineno": 204,
        "msg": "Tracker locked.",
        "pathname": "/home/pi/orangeandblueleague/filefairy/plugin/exports/exports.py"
      },
      {
        "count": 1,
        "date": "2018-08-30T20:16:42.865234",
        "exc": "",
        "levelname": "INFO",
        "lineno": 96,
        "msg": "Upload started.",
        "pathname": "/home/pi/orangeandblueleague/filefairy/plugin/leaguefile/leaguefile.py"
      }
    ],
    "2018-08-31T00:00:00": [
      {
        "count": 1,
        "date": "2018-08-31T00:01:55.843245",
        "exc": "",
        "levelname": "INFO",
        "lineno": 164,
        "msg": "Automated data push.",
        "pathname": "/home/pi/orangeandblueleague/filefairy/plugin/git/git.py"
      },
      {
        "count": 1,
        "date": "2018-08-31T03:44:52.130923",
        "exc": "",
        "levelname": "INFO",
        "lineno": 294,
        "msg": "File is up.",
        "pathname": "/home/pi/orangeandblueleague/filefairy/plugin/leaguefile/leaguefile.py"
      },
      {
        "count": 1,
        "date": "2018-08-31T03:44:55.533356",
        "exc": "",
        "levelname": "INFO",
        "lineno": 288,
        "msg": "Download started.",
        "pathname": "/home/pi/orangeandblueleague/filefairy/plugin/leaguefile/leaguefile.py"
      },
      {
        "count": 1,
        "date": "2018-08-31T03:50:10.344612",
        "exc": "",
        "levelname": "INFO",
        "lineno": 270,
        "msg": "Download finished.",
        "pathname": "/home/pi/orangeandblueleague/filefairy/plugin/leaguefile/leaguefile.py"
      },
      {
        "count": 1,
        "date": "2018-08-31T03:50:16.117535",
        "exc": "",
        "levelname": "INFO",
        "lineno": 60,
        "msg": "News updated.",
        "pathname": "/home/pi/orangeandblueleague/filefairy/plugin/recap/recap.py"
      }
    ],
    "2018-09-01T00:00:00": [
      {
        "count": 1,
        "date": "2018-09-01T00:02:06.397737",
        "exc": "",
        "levelname": "INFO",
        "lineno": 164,
        "msg": "Automated data push.",
        "pathname": "/home/pi/orangeandblueleague/filefairy/plugin/git/git.py"
      },
      {
        "count": 1,
        "date": "2018-09-01T00:36:45.880608",
        "exc": "Traceback (most recent call last):\n  File \"/usr/lib/python3.5/subprocess.py\", line 385, in run\n    stdout, stderr = process.communicate(input, timeout=timeout)\n  File \"/usr/lib/python3.5/subprocess.py\", line 801, in communicate\n    stdout, stderr = self._communicate(input, endtime, timeout)\n  File \"/usr/lib/python3.5/subprocess.py\", line 1447, in _communicate\n    self._check_timeout(endtime, orig_timeout)\n  File \"/usr/lib/python3.5/subprocess.py\", line 829, in _check_timeout\n    raise TimeoutExpired(self.args, orig_timeout)\nsubprocess.TimeoutExpired: Command '['scp', '/home/pi/orangeandblueleague/filefairy/resource/html/fairylab/exports/index.html', 'brunnerj@server:/var/www/html/fairylab/exports/index.html']' timed out after 8 seconds\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/home/pi/orangeandblueleague/filefairy/util/subprocess_/subprocess_.py\", line 18, in check_output\n    check=True)\n  File \"/usr/lib/python3.5/subprocess.py\", line 390, in run\n    stderr=stderr)\nsubprocess.TimeoutExpired: Command '['scp', '/home/pi/orangeandblueleague/filefairy/resource/html/fairylab/exports/index.html', 'brunnerj@server:/var/www/html/fairylab/exports/index.html']' timed out after 8 seconds",
        "levelname": "WARNING",
        "lineno": 24,
        "msg": "Handled warning.",
        "pathname": "/home/pi/orangeandblueleague/filefairy/util/subprocess_/subprocess_.py"
      },
      {
        "count": 1,
        "date": "2018-09-01T02:12:40.697264",
        "exc": "Traceback (most recent call last):\n  File \"/usr/lib/python3.5/subprocess.py\", line 385, in run\n    stdout, stderr = process.communicate(input, timeout=timeout)\n  File \"/usr/lib/python3.5/subprocess.py\", line 801, in communicate\n    stdout, stderr = self._communicate(input, endtime, timeout)\n  File \"/usr/lib/python3.5/subprocess.py\", line 1447, in _communicate\n    self._check_timeout(endtime, orig_timeout)\n  File \"/usr/lib/python3.5/subprocess.py\", line 829, in _check_timeout\n    raise TimeoutExpired(self.args, orig_timeout)\nsubprocess.TimeoutExpired: Command '['scp', '/home/pi/orangeandblueleague/filefairy/resource/html/fairylab/exports/index.html', 'brunnerj@server:/var/www/html/fairylab/exports/index.html']' timed out after 8 seconds\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/home/pi/orangeandblueleague/filefairy/util/subprocess_/subprocess_.py\", line 18, in check_output\n    check=True)\n  File \"/usr/lib/python3.5/subprocess.py\", line 390, in run\n    stderr=stderr)\nsubprocess.TimeoutExpired: Command '['scp', '/home/pi/orangeandblueleague/filefairy/resource/html/fairylab/exports/index.html', 'brunnerj@server:/var/www/html/fairylab/exports/index.html']' timed out after 8 seconds",
        "levelname": "WARNING",
        "lineno": 24,
        "msg": "Handled warning.",
        "pathname": "/home/pi/orangeandblueleague/filefairy/util/subprocess_/subprocess_.py"
      },
      {
        "count": 1,
        "date": "2018-09-01T06:26:46.340108",
        "exc": "Traceback (most recent call last):\n  File \"/usr/lib/python3.5/subprocess.py\", line 385, in run\n    stdout, stderr = process.communicate(input, timeout=timeout)\n  File \"/usr/lib/python3.5/subprocess.py\", line 801, in communicate\n    stdout, stderr = self._communicate(input, endtime, timeout)\n  File \"/usr/lib/python3.5/subprocess.py\", line 1447, in _communicate\n    self._check_timeout(endtime, orig_timeout)\n  File \"/usr/lib/python3.5/subprocess.py\", line 829, in _check_timeout\n    raise TimeoutExpired(self.args, orig_timeout)\nsubprocess.TimeoutExpired: Command '['scp', '/home/pi/orangeandblueleague/filefairy/resource/html/fairylab/exports/index.html', 'brunnerj@server:/var/www/html/fairylab/exports/index.html']' timed out after 8 seconds\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/home/pi/orangeandblueleague/filefairy/util/subprocess_/subprocess_.py\", line 18, in check_output\n    check=True)\n  File \"/usr/lib/python3.5/subprocess.py\", line 390, in run\n    stderr=stderr)\nsubprocess.TimeoutExpired: Command '['scp', '/home/pi/orangeandblueleague/filefairy/resource/html/fairylab/exports/index.html', 'brunnerj@server:/var/www/html/fairylab/exports/index.html']' timed out after 8 seconds",
        "levelname": "WARNING",
        "lineno": 24,
        "msg": "Handled warning.",
        "pathname": "/home/pi/orangeandblueleague/filefairy/util/subprocess_/subprocess_.py"
      },
      {
        "count": 1,
        "date": "2018-09-01T10:00:07.046597",
        "exc": "Traceback (most recent call last):\n  File \"/usr/lib/python3.5/subprocess.py\", line 385, in run\n    stdout, stderr = process.communicate(input, timeout=timeout)\n  File \"/usr/lib/python3.5/subprocess.py\", line 801, in communicate\n    stdout, stderr = self._communicate(input, endtime, timeout)\n  File \"/usr/lib/python3.5/subprocess.py\", line 1447, in _communicate\n    self._check_timeout(endtime, orig_timeout)\n  File \"/usr/lib/python3.5/subprocess.py\", line 829, in _check_timeout\n    raise TimeoutExpired(self.args, orig_timeout)\nsubprocess.TimeoutExpired: Command '['scp', '/home/pi/orangeandblueleague/filefairy/resource/html/fairylab/index.html', 'brunnerj@server:/var/www/html/fairylab/index.html']' timed out after 8 seconds\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/home/pi/orangeandblueleague/filefairy/util/subprocess_/subprocess_.py\", line 18, in check_output\n    check=True)\n  File \"/usr/lib/python3.5/subprocess.py\", line 390, in run\n    stderr=stderr)\nsubprocess.TimeoutExpired: Command '['scp', '/home/pi/orangeandblueleague/filefairy/resource/html/fairylab/index.html', 'brunnerj@server:/var/www/html/fairylab/index.html']' timed out after 8 seconds",
        "levelname": "WARNING",
        "lineno": 24,
        "msg": "Handled warning.",
        "pathname": "/home/pi/orangeandblueleague/filefairy/util/subprocess_/subprocess_.py"
      },
      {
        "count": 1,
        "date": "2018-09-01T10:17:30.060171",
        "exc": "Traceback (most recent call last):\n  File \"/usr/lib/python3.5/subprocess.py\", line 385, in run\n    stdout, stderr = process.communicate(input, timeout=timeout)\n  File \"/usr/lib/python3.5/subprocess.py\", line 801, in communicate\n    stdout, stderr = self._communicate(input, endtime, timeout)\n  File \"/usr/lib/python3.5/subprocess.py\", line 1447, in _communicate\n    self._check_timeout(endtime, orig_timeout)\n  File \"/usr/lib/python3.5/subprocess.py\", line 829, in _check_timeout\n    raise TimeoutExpired(self.args, orig_timeout)\nsubprocess.TimeoutExpired: Command '['scp', '/home/pi/orangeandblueleague/filefairy/resource/html/fairylab/exports/index.html', 'brunnerj@server:/var/www/html/fairylab/exports/index.html']' timed out after 8 seconds\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/home/pi/orangeandblueleague/filefairy/util/subprocess_/subprocess_.py\", line 18, in check_output\n    check=True)\n  File \"/usr/lib/python3.5/subprocess.py\", line 390, in run\n    stderr=stderr)\nsubprocess.TimeoutExpired: Command '['scp', '/home/pi/orangeandblueleague/filefairy/resource/html/fairylab/exports/index.html', 'brunnerj@server:/var/www/html/fairylab/exports/index.html']' timed out after 8 seconds",
        "levelname": "WARNING",
        "lineno": 24,
        "msg": "Handled warning.",
        "pathname": "/home/pi/orangeandblueleague/filefairy/util/subprocess_/subprocess_.py"
      },
      {
        "count": 1,
        "date": "2018-09-01T11:21:01.575759",
        "exc": "Traceback (most recent call last):\n  File \"/usr/lib/python3.5/subprocess.py\", line 385, in run\n    stdout, stderr = process.communicate(input, timeout=timeout)\n  File \"/usr/lib/python3.5/subprocess.py\", line 801, in communicate\n    stdout, stderr = self._communicate(input, endtime, timeout)\n  File \"/usr/lib/python3.5/subprocess.py\", line 1447, in _communicate\n    self._check_timeout(endtime, orig_timeout)\n  File \"/usr/lib/python3.5/subprocess.py\", line 829, in _check_timeout\n    raise TimeoutExpired(self.args, orig_timeout)\nsubprocess.TimeoutExpired: Command '['scp', '/home/pi/orangeandblueleague/filefairy/resource/html/fairylab/exports/index.html', 'brunnerj@server:/var/www/html/fairylab/exports/index.html']' timed out after 8 seconds\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/home/pi/orangeandblueleague/filefairy/util/subprocess_/subprocess_.py\", line 18, in check_output\n    check=True)\n  File \"/usr/lib/python3.5/subprocess.py\", line 390, in run\n    stderr=stderr)\nsubprocess.TimeoutExpired: Command '['scp', '/home/pi/orangeandblueleague/filefairy/resource/html/fairylab/exports/index.html', 'brunnerj@server:/var/www/html/fairylab/exports/index.html']' timed out after 8 seconds",
        "levelname": "WARNING",
        "lineno": 24,
        "msg": "Handled warning.",
        "pathname": "/home/pi/orangeandblueleague/filefairy/util/subprocess_/subprocess_.py"
      },
      {
        "count": 1,
        "date": "2018-09-01T12:05:04.527447",
        "exc": "Traceback (most recent call last):\n  File \"/usr/lib/python3.5/subprocess.py\", line 385, in run\n    stdout, stderr = process.communicate(input, timeout=timeout)\n  File \"/usr/lib/python3.5/subprocess.py\", line 801, in communicate\n    stdout, stderr = self._communicate(input, endtime, timeout)\n  File \"/usr/lib/python3.5/subprocess.py\", line 1447, in _communicate\n    self._check_timeout(endtime, orig_timeout)\n  File \"/usr/lib/python3.5/subprocess.py\", line 829, in _check_timeout\n    raise TimeoutExpired(self.args, orig_timeout)\nsubprocess.TimeoutExpired: Command '['ssh', 'brunnerj@server', 'ls -l /var/www/html/StatsLab/league_file']' timed out after 8 seconds\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/home/pi/orangeandblueleague/filefairy/util/subprocess_/subprocess_.py\", line 18, in check_output\n    check=True)\n  File \"/usr/lib/python3.5/subprocess.py\", line 390, in run\n    stderr=stderr)\nsubprocess.TimeoutExpired: Command '['ssh', 'brunnerj@server', 'ls -l /var/www/html/StatsLab/league_file']' timed out after 8 seconds",
        "levelname": "WARNING",
        "lineno": 24,
        "msg": "Handled warning.",
        "pathname": "/home/pi/orangeandblueleague/filefairy/util/subprocess_/subprocess_.py"
      },
      {
        "count": 1,
        "date": "2018-09-01T12:18:08.494064",
        "exc": "Traceback (most recent call last):\n  File \"/usr/lib/python3.5/subprocess.py\", line 385, in run\n    stdout, stderr = process.communicate(input, timeout=timeout)\n  File \"/usr/lib/python3.5/subprocess.py\", line 801, in communicate\n    stdout, stderr = self._communicate(input, endtime, timeout)\n  File \"/usr/lib/python3.5/subprocess.py\", line 1447, in _communicate\n    self._check_timeout(endtime, orig_timeout)\n  File \"/usr/lib/python3.5/subprocess.py\", line 829, in _check_timeout\n    raise TimeoutExpired(self.args, orig_timeout)\nsubprocess.TimeoutExpired: Command '['scp', '/home/pi/orangeandblueleague/filefairy/resource/html/fairylab/exports/index.html', 'brunnerj@server:/var/www/html/fairylab/exports/index.html']' timed out after 8 seconds\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/home/pi/orangeandblueleague/filefairy/util/subprocess_/subprocess_.py\", line 18, in check_output\n    check=True)\n  File \"/usr/lib/python3.5/subprocess.py\", line 390, in run\n    stderr=stderr)\nsubprocess.TimeoutExpired: Command '['scp', '/home/pi/orangeandblueleague/filefairy/resource/html/fairylab/exports/index.html', 'brunnerj@server:/var/www/html/fairylab/exports/index.html']' timed out after 8 seconds",
        "levelname": "WARNING",
        "lineno": 24,
        "msg": "Handled warning.",
        "pathname": "/home/pi/orangeandblueleague/filefairy/util/subprocess_/subprocess_.py"
      },
      {
        "count": 1,
        "date": "2018-09-01T14:09:07.493525",
        "exc": "Traceback (most recent call last):\n  File \"/usr/lib/python3.5/subprocess.py\", line 385, in run\n    stdout, stderr = process.communicate(input, timeout=timeout)\n  File \"/usr/lib/python3.5/subprocess.py\", line 801, in communicate\n    stdout, stderr = self._communicate(input, endtime, timeout)\n  File \"/usr/lib/python3.5/subprocess.py\", line 1447, in _communicate\n    self._check_timeout(endtime, orig_timeout)\n  File \"/usr/lib/python3.5/subprocess.py\", line 829, in _check_timeout\n    raise TimeoutExpired(self.args, orig_timeout)\nsubprocess.TimeoutExpired: Command '['scp', '/home/pi/orangeandblueleague/filefairy/resource/html/fairylab/snacks/index.html', 'brunnerj@server:/var/www/html/fairylab/snacks/index.html']' timed out after 8 seconds\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/home/pi/orangeandblueleague/filefairy/util/subprocess_/subprocess_.py\", line 18, in check_output\n    check=True)\n  File \"/usr/lib/python3.5/subprocess.py\", line 390, in run\n    stderr=stderr)\nsubprocess.TimeoutExpired: Command '['scp', '/home/pi/orangeandblueleague/filefairy/resource/html/fairylab/snacks/index.html', 'brunnerj@server:/var/www/html/fairylab/snacks/index.html']' timed out after 8 seconds",
        "levelname": "WARNING",
        "lineno": 24,
        "msg": "Handled warning.",
        "pathname": "/home/pi/orangeandblueleague/filefairy/util/subprocess_/subprocess_.py"
      },
      {
        "count": 1,
        "date": "2018-09-01T15:16:32.945238",
        "exc": "",
        "levelname": "INFO",
        "lineno": 204,
        "msg": "Tracker locked.",
        "pathname": "/home/pi/orangeandblueleague/filefairy/plugin/exports/exports.py"
      },
      {
        "count": 2,
        "date": "2018-09-01T15:51:48.198822",
        "exc": "Traceback (most recent call last):\n  File \"/usr/lib/python3.5/subprocess.py\", line 385, in run\n    stdout, stderr = process.communicate(input, timeout=timeout)\n  File \"/usr/lib/python3.5/subprocess.py\", line 801, in communicate\n    stdout, stderr = self._communicate(input, endtime, timeout)\n  File \"/usr/lib/python3.5/subprocess.py\", line 1447, in _communicate\n    self._check_timeout(endtime, orig_timeout)\n  File \"/usr/lib/python3.5/subprocess.py\", line 829, in _check_timeout\n    raise TimeoutExpired(self.args, orig_timeout)\nsubprocess.TimeoutExpired: Command '['scp', '/home/pi/orangeandblueleague/filefairy/resource/html/fairylab/exports/index.html', 'brunnerj@server:/var/www/html/fairylab/exports/index.html']' timed out after 8 seconds\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/home/pi/orangeandblueleague/filefairy/util/subprocess_/subprocess_.py\", line 18, in check_output\n    check=True)\n  File \"/usr/lib/python3.5/subprocess.py\", line 390, in run\n    stderr=stderr)\nsubprocess.TimeoutExpired: Command '['scp', '/home/pi/orangeandblueleague/filefairy/resource/html/fairylab/exports/index.html', 'brunnerj@server:/var/www/html/fairylab/exports/index.html']' timed out after 8 seconds",
        "levelname": "WARNING",
        "lineno": 24,
        "msg": "Handled warning.",
        "pathname": "/home/pi/orangeandblueleague/filefairy/util/subprocess_/subprocess_.py"
      },
      {
        "count": 1,
        "date": "2018-09-01T16:39:54.031368",
        "exc": "Traceback (most recent call last):\n  File \"/usr/lib/python3.5/subprocess.py\", line 385, in run\n    stdout, stderr = process.communicate(input, timeout=timeout)\n  File \"/usr/lib/python3.5/subprocess.py\", line 801, in communicate\n    stdout, stderr = self._communicate(input, endtime, timeout)\n  File \"/usr/lib/python3.5/subprocess.py\", line 1447, in _communicate\n    self._check_timeout(endtime, orig_timeout)\n  File \"/usr/lib/python3.5/subprocess.py\", line 829, in _check_timeout\n    raise TimeoutExpired(self.args, orig_timeout)\nsubprocess.TimeoutExpired: Command '['scp', '/home/pi/orangeandblueleague/filefairy/resource/html/fairylab/exports/index.html', 'brunnerj@server:/var/www/html/fairylab/exports/index.html']' timed out after 8 seconds\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/home/pi/orangeandblueleague/filefairy/util/subprocess_/subprocess_.py\", line 18, in check_output\n    check=True)\n  File \"/usr/lib/python3.5/subprocess.py\", line 390, in run\n    stderr=stderr)\nsubprocess.TimeoutExpired: Command '['scp', '/home/pi/orangeandblueleague/filefairy/resource/html/fairylab/exports/index.html', 'brunnerj@server:/var/www/html/fairylab/exports/index.html']' timed out after 8 seconds",
        "levelname": "WARNING",
        "lineno": 24,
        "msg": "Handled warning.",
        "pathname": "/home/pi/orangeandblueleague/filefairy/util/subprocess_/subprocess_.py"
      }
    ]
  }
}
